src/test/kotlin/study/CalculatorValidTest.kt

```diff
@@ -0,0 +1,73 @@
+package study
+
+import calculator.ValidCheck
+import calculator.calculateOperationList
+import calculator.operation.Addition
+import org.assertj.core.api.Assertions.assertThatThrownBy
+import org.junit.jupiter.api.Assertions.assertTrue
+import org.junit.jupiter.api.DisplayName
+import org.junit.jupiter.api.Test
+import org.junit.jupiter.api.TestInstance
+import org.junit.jupiter.params.ParameterizedTest
+import org.junit.jupiter.params.provider.Arguments
+import org.junit.jupiter.params.provider.EmptySource
+import org.junit.jupiter.params.provider.MethodSource
+import org.junit.jupiter.params.provider.NullSource
+import org.junit.jupiter.params.provider.ValueSource
+import java.util.stream.Stream
+
+
+@TestInstance(TestInstance.Lifecycle.PER_CLASS)
+class CalculatorValidTest {
+
+    @Test
+    @DisplayName("문자열 split 배열 개수 유효성 검사")
+    fun splitValidCount() {
```

Comment:
코틀린에서 테스트코드의 이름은 `@DisplayName`을 쓰지 않고, 다음과 같이 작성하는 편이에요.
```suggestion
    fun `문자열 split 배열 개수 유효성 검사`() {
```